File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);
    
    int arr[N];
    
    // N개의 정수를 입력받아 배열에 저장
    for (int i = 0; i < N; i++) {
        scanf("%d", &arr[i]);
    }

    int min = arr[0]; 
    int max = arr[0];

    // 배열을 순회하면서 최솟값과 최댓값 찾기
    for (int j = 0; j < N; j++) {
        // 현재 값이 최솟값보다 작으면 갱신
        if (arr[j] < min) {
            min = arr[j];
        }
        // 현재 값이 최댓값보다 크면 갱신
        if (arr[j] > max) {
            max = arr[j];
        }
    }

    printf("%d %d", min, max);

    return 0;
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf8').trim().split(/\s+/).map(Number);

let min = input[1];
let max = input[1];

for (let i = 2; i < input.length; i++) {
  const v = input[i];
  if (v < min) min = v;
  if (v > max) max = v;
}

console.log(min, max);